src/position.cpp

```diff
@@ -1546,10 +1559,28 @@ Value Position::see(Move m) const {
 #endif
 
       // Add the new entry to the swap list
+#ifdef ANTI
+      if (is_anti())
+          swapList[slIndex] = -swapList[slIndex - 1] + PieceValueAnti[MG][captured];
+      else
+#endif
       swapList[slIndex] = -swapList[slIndex - 1] + PieceValue[MG][captured];
 
       // Locate and remove the next least valuable attacker
       captured = min_attacker<PAWN>(byTypeBB, to, stmAttackers, occupied, attackers);
+#ifdef ANTI
```

Comment:
The point is that `min_attacker<KING>` does not update `occupied` and `attackers` bitboards, because SEE always stops before king captures in standard chess. However, in Antichess we need the the updated bitboards, and since `min_attacker` does not get the position as an argument, I decided to add the code snippet to the SEE. Feel free to move this to `min_attacker` (and add the position as an argument), if you prefer to have it there, although we would probably have to make sure that it does not give a slow-down.